services/core-api/app/api/users/minespace/resources/minespace_user.py

```diff
@@ -63,3 +68,37 @@ def delete(self, user_id):
         db.session.delete(user)
         db.session.commit()
         return ('', 204)
+
+    @api.doc(description='Update an existing Minespace Users mine list')
+    @api.marshal_with(MINESPACE_USER_MODEL, envelope='records')
+    @requires_role_mine_admin
+    def put(self, user_id):
+        # current_app.logger.debug('Mine list found, checking for new mines')
+        contact = MinespaceUser.find_by_id(user_id)
```

Comment:
remove logger

Reply:
if you're using current_app and logging to info, warning, and so on its probably fine to leave them in its just debug that I would remove.